src/components/quickStarts.stories.mdx

```diff
@@ -509,10 +509,11 @@ const MyApp = () => {
   const joinMeeting = async () => {
     const response = await fetch('/my-meetings-endpoint');
     const data = await response.json();
+    const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
+    const eventController = new DefaultEventController(meetingSessionConfiguration, meetingManager.createLogger(meetingSessionConfiguration), new NoOpEventReporter())
```

Comment:
Can fix later or in quick follow up: Can we prettify these examples? Not sure how this looks in storybook.

Reply:
Updated